File: backend/BENewsFeed/BENewsFeed/dto/ArticleDto.cs
namespace BENewsFeed.dto
{
    public class ArticleDto
    {
        public int Id { get; set; }
        public string Title { get; set; } = "";
        public string Description { get; set; } = "";
        public string ImageUrl { get; set; } = "";
        public string Link { get; set; } = "";
        public string PublishedAt { get; set; } = "";
        public string SourceName { get; set; } = "";
        public string Category { get; set; } = "";
    }
}

File: android/app/src/main/java/com/example/newsfeed/data/remote/dto/ArticleDto.kt
package com.example.newsfeed.data.remote.dto

import com.google.gson.annotations.SerializedName

data class ArticleDto (

    val id: Int,
    val title: String,
    val description: String,
    @SerializedName("imageUrl")
    val imageUrl: String,
    val link: String,
    val publishedAt: String,
    val sourceName: String,
    val category: String
)